.gitignore

```diff
@@ -1 +1,4 @@
-.idea/
\ No newline at end of file
+.idea/
```

Comment:
Her kan du med fordel benytte "standard" python .gitignore, sjå feks i github.com/Informasjosforvaltning/concepttordf.git

Reply:
Ser jeg trengte å rydde litt i .gitignore fila, men skjønner ikke hvorfor jeg skal bruke en kjempelang standard fil når det meste av det som ligger i den ikke er i bruk i prosjektet?

Reply:
Nyttig for meg for å gi meg eit hint når nye artefakter dukkar opp ved bruk av verktøy. Men ikkje big deal.  Ignorer kommentaren om du vil.